pkg/vmimage/data/script.sh

```diff
@@ -157,6 +158,7 @@ yum -y install \
     rootfiles \
     rsyslog \
     samba-client \
+    screen \
```

Comment:
scap needs screen? O_o I would be tempted to remove it with the argument - no ssh - no screen needed. @jim-minter wdyt?

Reply:
Removed screen.